cms/djangoapps/contentstore/rest_api/v1/views/grading.py

```diff
@@ -85,7 +85,8 @@ def get(self, request: Request, course_id: str):
                 "minimum_grade_credit": 0.7
             },
             "show_credit_eligibility": false,
-            "is_credit_course": true
+            "is_credit_course": true,
+            "default_grade_designations": ["A","B","C","D"]
```

Comment:
Hey @kaustavb12  we might need to add this in the serializer as well right? In case we want to push it out in the API response. I have made a GET request to ``http://localhost:18010/api/contentstore/v1/course_grading/{course_id}`` but I don't get back the default_grade_designation with the PR. I ended up adding:

```
diff --git a/cms/djangoapps/contentstore/rest_api/v1/serializers/grading.py b/cms/djangoapps/contentstore/rest_api/v1/serializers/grading.py
index 1ff8b794f8..5b320e738e 100644
--- a/cms/djangoapps/contentstore/rest_api/v1/serializers/grading.py
+++ b/cms/djangoapps/contentstore/rest_api/v1/serializers/grading.py
@@ -40,3 +40,4 @@ class CourseGradingSerializer(serializers.Serializer):
     course_details = CourseGradingModelSerializer()
     show_credit_eligibility = serializers.BooleanField()
     is_credit_course = serializers.BooleanField()
+    default_grade_designations = serializers.ListSerializer(child=serializers.CharField())
diff --git a/cms/envs/devstack-experimental.yml b/cms/envs/devstack-experimental.yml
index 54f6edf8f9..2a0d6467d9 100644
--- a/cms/envs/devstack-experimental.yml
+++ b/cms/envs/devstack-experimental.yml
@@ -296,6 +296,7 @@ EXTRA_MIDDLEWARE_CLASSES: []
 FACEBOOK_API_VERSION: v2.1
 FACEBOOK_APP_ID: FACEBOOK_APP_ID
 FACEBOOK_APP_SECRET: FACEBOOK_APP_SECRET
+DEFAULT_GRADE_DESIGNATIONS: ['A+']
 FEATURES:
     AUTH_USE_OPENID_PROVIDER: true
     AUTOMATIC_AUTH_FOR_TESTING: false
```

Maybe I am missing something.

Reply:
@farhaanbukhsh Thanks for catching this. The serializer was implemented after this PR was created.
But the missing entry in the serializer should have been added during the last rebase, which is my bad.